assembl/configs/base_env.yaml

```diff
@@ -186,3 +186,9 @@ attachment_allowed_mime_types:
   - image/*
   - video/*
   - application/pdf
+  - application/msword
+  - application/vnd.openxmlformats-officedocument.wordprocessingml.document
```

Comment:
You could also do a regex which will capture all of the list that you've just added, including perhaps other ms or office types that you may have missed:
```
r"application\/vnd\.(.*ms.*|.*officedocument.*)"
```

Reply:
But you'll have to make sure that the check downstream does a regex check

Reply:
Good point, I'll implement this to get a shorter list.